framework/derive/src/parse/attributes/util.rs

```diff
@@ -12,6 +15,66 @@ pub(super) fn is_attribute_with_no_args(attr: &syn::Attribute, name: &str) -> bo
     false
 }
 
+pub(super) fn get_attribute_with_one_type_arg(
+    attr: &syn::Attribute,
+    name: &str,
+) -> Option<EsdtAttribute> {
+    let attr_path = &attr.path;
+    if let Some(first_seg) = attr_path.segments.first() {
+        if first_seg.ident == name {
+            let tokens = attr.tokens.clone();
+            let group = match syn::parse2::<Group>(tokens) {
```

Comment:
Do we need to _parse_ the tokens?
I would expect an `if let Group(g) = tokens ... ` should be enough.